jaggr-core/pom.xml

```diff
@@ -303,7 +303,7 @@
               <directory>.</directory>
               <includes>
                 <include>lib/**</include>
-                <include>src/main/resources/less*.js</include>
+                <include>src/main/resources/less-rhino-*.js</include>
```

Comment:
The directory now contains the file lessFileLoader.js so I had to make the delete selector more specific.